test/sqs_test_case.py

```diff
@@ -61,10 +63,22 @@ def _read_queue(self, queue) -> MutableJSONs:
         message_bodies = [json.loads(m.body) for m in messages]
         return message_bodies
 
-    def _mock_sqs_record(self, body, *, attempts: int = 1):
+    def _mock_sqs_record(self,
+                         body: JSON,
+                         *,
+                         attempts: int = 1,
+                         fifo: bool = False,
+                         ) -> SQSRecord:
         event_dict = {
+            'messageId': 'ThisWasARandomUUID',
             'body': json.dumps(body),
             'receiptHandle': 'ThisWasARandomString',
-            'attributes': {'ApproximateReceiveCount': attempts}
+            'attributes': {
+                'ApproximateReceiveCount': str(attempts),
+                **iif(fifo, {
+                    'MessageGroupId': 'AlsoARandomUUID',
+                    'MessageDeduplicationId': 'AndAnotherRandomUUID'
```

Comment:
The same IDs for every message? That might make diagnosing tests more difficult, because log lines for different messages would contain the same ID. Consider using a seeded random generator to generate these UUIDs.